scripts/build.ps1

```diff
@@ -569,6 +569,17 @@ function Publish-Package
 
     Copy-Item -Recurse $intellitraceSourceDirectory\* $intellitraceTargetDirectory -Force
     
+    # Copy Microsoft.VisualStudio.TraceDataCollector
+    $codeCoverageExternalsVersion = ([xml](Get-Content $env:TP_ROOT_DIR\scripts\build\TestPlatform.Dependencies.props)).Project.PropertyGroup.CodeCoverageExternalsVersion
+    $traceDataCollectorSourceDirectory = Join-Path $env:TP_PACKAGES_DIR "Microsoft.VisualStudio.TraceDataCollector\$codeCoverageExternalsVersion\lib"
+    $traceDataCollectorTargetDirectory = Join-Path $env:TP_OUT_DIR "$TPB_Configuration\Microsoft.VisualStudio.TraceDataCollector"
```

Comment:
what is the purpose of this copy? Wondering why target contains folder Microsoft.VisualStudio.TraceDataCollector
we don't have such folder in vsix or nuget

Reply:
This copies libs from nuget package to artifacts folder, and then nuspec file gets it from there,

Reply:
did you check vsix if all tracedatacollector is correctly there?

Reply:
yes i have,